hsv-dot-beer/hsvdotbeer
# Use our own version of the Stem and Stein logo

I took the logo and put it on a 1024x1024 black square so that way it'll show up no matter what.

Fixes #251.

To test locally:

`pipenv run ./manage.py collectstatic`

Observe that you now have `static/venues/png/stem-and-stein-black.png` in your tree.

```bash
drew@sirrobin:/c/Users/drew/git/hsvdotbeer$ ls static/venues/png/
stem-and-stein-black.png
```
